.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
Yeah, true. 
I still don't like having arc descriptions in one of the RPs; it's functional, but it's going to bite people later when they can't remember which RP the description is in.

I wanted to avoid making canons "arcs again, but different"; given that we generate a page for every single tag, generating one for every arc isn't that absurd.